docs/pages/getting-started.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Getting Started
+layout: default
+permalink: /getting-started
+nav_order: 2
+---
+
+## Table of Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+---
+
+# Getting Started
+
+Getting started requires a few steps:
+There are a number of steps that must be completed when first getting started with Stabilizer.
+1. Configure the firmware
+    * This requires updating any parameters, such as static IP addresses and
+   sampling rate.
+1. Build the application
+    * This requires compiling the code after configuration parameters have been
+   updated.
+1. Upload the application
+    * Once fimrware has been built, it needs to be programmed onto the device.
+1. Set up MQTT
+    * Stabilizer utilizes MQTT for telemetry and configuration.
+
+The following sections will walk you through completing each of these steps.
+
+# Program Stabilizer
+
+Firmware is compiled and loaded onto Stabilizer to program a specific application.
+
+After receiving the Stabilizer hardware, you will need to flash firmware onto the device to use your
+desired application.
+
+## Configuring Firmware
+
+Stabilizer firmware contains compile-time parameters that may need to be changed based on
+application requirements. Some examples of parameters that may require configuraton:
+* Sample frequency
+* Sample batch size
+* MQTT Broker IP address
+
+Parameters are configured by editing `src/configuration.rs`.
+
+Refer to the [documentation]({{site.baseurl}}/firmware/stabilizer/configuration.rs) for more
```

Comment:
To catch dead links like [this one](https://quartiq.de/stabilizer/firmware/stabilizer/configuration.rs), maybe [htmlproofer](https://clementbm.github.io/github%20action/jekyll/link%20checking/2020/05/31/automatically-validate-links-on-jekyll-website.html)?

Reply:
There's some others as well that refer to `/#application` instead of `{{site.baseurl}}/#application`. Definitely need to look for a tool to proof the Jekyll output